packages/react/src/components/InlineAlert/index.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withKnobs, select, text } from "@storybook/addon-knobs";
+import { withInfo } from "@storybook/addon-info";
+
+import InlineAlert from "./";
+import { INLINE_ALERT_TYPES } from "../../constants/";
+
+storiesOf("Alerts")
```

Comment:
So that was my idea: :)

Since the amount of components is growing, I was thinking about grouping them and putting all alerts into `Alerts`. But maybe it's too early to do that :)

Reply:
What suits you the best 👌